src/System.Runtime.CompilerServices.VisualC/ref/System.Runtime.CompilerServices.VisualC.manual.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+[assembly:System.Runtime.CompilerServices.InternalsVisibleTo("mscorlib, PublicKey=00000000000000000400000000000000")]
+
+namespace System.Runtime.CompilerServices
+{
+    [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    internal sealed class AssemblyAttributesGoHere
+    {
+        internal AssemblyAttributesGoHere()
+        {
+        }
+    }
+    [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    internal sealed class AssemblyAttributesGoHereS
+    {
+        internal AssemblyAttributesGoHereS()
+        {
+        }
+    }
+    [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    internal sealed class AssemblyAttributesGoHereM
+    {
+        internal AssemblyAttributesGoHereM()
+        {
+        }
+    }
+    [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    internal sealed class AssemblyAttributesGoHereSM
+    {
+        internal AssemblyAttributesGoHereSM()
+        {
+        }
+    }
+    [System.AttributeUsage(System.AttributeTargets.All)]
+    internal sealed class DecoratedNameAttribute : System.Attribute
+    {
+        public DecoratedNameAttribute(string decoratedName) {}
+    }
+    [AttributeUsage(AttributeTargets.Class |
+                    AttributeTargets.Constructor |
+                    AttributeTargets.Method |
+                    AttributeTargets.Field |
+                    AttributeTargets.Event |
+                    AttributeTargets.Property)]
+    [TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    internal sealed class SuppressMergeCheckAttribute : Attribute
+    {
+        public SuppressMergeCheckAttribute()
```

Comment:
It doesn't really matter, but how come some of these internal attributes have their ctors defined as public rather than internal?  Can the explicit ctors just be deleted entirely and let the compiler provide the default one?

Reply:
I just copied the definitions straight from the reference source. I can remove them if you want.

Reply:
No need to rock the boat.  Might want to clean it up later, though.